### PR TITLE
[cinder-csi-plugin] Updated CLOUD_CONFIG in Helm Chart

### PR DESCRIPTION
**Pull request by powellchristoph**
**What this PR does / why we need it**: 

Currently the openstack-ccm helm chart [hardcodes its key name](https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/openstack-cloud-controller-manager/templates/secret.yaml#L8) when creating its secret as `cloud.conf`. Due to the different secret keys, you must create two separate secrets if running both the CCM and CSI plugin. This change would allow the `cinder-csi-plugin` reuse the existing secret created by the CCM.

```release-note
* Changed CLOUD_CONFIG env var in Helm Chart.
```

### CODE REVIEW

**Comment by powellchristoph** [comment]
Completed CLA Authorization.

**Comment by jichenjc** [comment]
I don't remember clearly whether cloud.conf is same in CSI and OCCM
if they do, yes, we need think about make them identical

**Comment by powellchristoph** [comment]
Just a quick look at the source, it appears that each plugin references their own section in the config file so they should be fine.

**Comment by jichenjc** [comment]
could you please help to fix the CLA and other issues?

I think it's ok and reasonable to make the secret reusable if we can

File: charts/cinder-csi-plugin/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: latest
 description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 2.0.0
+version: 2.0.1
```
Comment (lingxiankong) [inline charts/cinder-csi-plugin/Chart.yaml:5]: Version bump will be done during the release, so please keep the version here.

**Review by lingxiankong — CHANGES_REQUESTED** [review]
and please fix the CLA issue.

**Comment by ramineni** [comment]
@powellchristoph I think the issue 2 commits are done with diff usernames. fixing should solve the cla issue

**Comment by powellchristoph** [comment]
I borked this all up. I'll submit a new one thats proper.

**Pull request closed without merging**